### DISCUSSION TITLE
Official Docker image: Please upgrade the version of Node.js used to run Renovate to satisfy ">= 18.12.0"

### DISCUSSION THREAD

**Discussion by szpak** [renovatebot/renovate#20335] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.129.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

When I execute Renovate using the latest official Docker image ([34.129.0](https://hub.docker.com/layers/renovate/renovate/34.129/images/sha256-ee31204a600d5f6a881e6800f6c3f4468e43f9b6c0df782183dd9e0397b2cccd?context=explore)), I see the following warning:

```
 INFO: Please upgrade the version of Node.js used to run Renovate to satisfy ">= 18.12.0". Support for your current version will be removed in Renovate's next major release.
       "versions": {
         "node": "16.19.0",
         "v8": "9.4.146.26-node.24",
         "uv": "1.43.0",
         "zlib": "1.2.11",
         "brotli": "1.0.9",
         "ares": "1.18.1",
         "modules": "93",
         "nghttp2": "1.47.0",
         "napi": "8",
         "llhttp": "6.0.10",
         "openssl": "1.1.1s+quic",
         "cldr": "41.0",
         "icu": "71.1",
         "tz": "2022f",
         "unicode": "14.0",
         "ngtcp2": "0.8.1",
         "nghttp3": "0.7.0"
       }
 INFO: Repository started (repository=group/repo)
       "renovateVersion": "34.129.0"
```

Previously, in some quote old Renovate version, I didn't see that warning.

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by HonkingGoose** [comment]
Should this issue be moved to: https://github.com/renovatebot/docker-renovate-full maybe? I'm not familiar with our Docker image publishing repositories, so I'll let the maintainers decide what to do here.

Maybe this is the file we need to change:

https://github.com/renovatebot/docker-renovate-full/blob/6ff1379cfbeb591078fd64f656a95cb65d9afa7c/Dockerfile#L13-L14

---

Edit: I found this PR that _may_ fix the problem:

- https://github.com/renovatebot/docker-renovate-full/pull/1323

Looks like we're waiting for `v35` to land before bumping the version of Node to 18?

**Comment by viceice** [comment, marked as answer]
just a info message, ignore it. renovate will soon make a major bump an then the images will switch to node v18. so the message will be gone.

**Reply by szpak** [reply]
OK. It just looked odd to see that message in the official Renovate image :D.

**Reply by viceice** [reply]
i know, but that's a tradeoff to not need an additional major release. 🙃
